### DISCUSSION TITLE
[Extended Sync] Unable to use Azure Workload Identity with official 0.7.0 charts.

### DISCUSSION THREAD

**Discussion by maciej-umanski** [eclipse-tractusx/tractusx-edc#1298] [Q&A]
I'm trying to deploy EDC using "tractusx-connector-azure-vault" on Azure using workload identity. While using official charts from release/0.7.0 the values that are important for automatic authorization to work are overritten by the Charts.

file: deployment-controlplane.yaml, lines: 319-322
file: deployment-dataplane.yaml, lines: 191-194
<img width="821" alt="image" src="https://github.com/eclipse-tractusx/tractusx-edc/assets/58386912/b56dc2f8-7aca-4547-9c1e-3c0ad11ea00f">

Here are the values in kubernetes pod:
<img width="644" alt="image" src="https://github.com/eclipse-tractusx/tractusx-edc/assets/58386912/416d5ad3-b84d-4390-a9d0-b45956094709">

If I manually edit the charts and make these 2 values optional, webhook is able to automatically assign these values to pods.
<img width="767" alt="image" src="https://github.com/eclipse-tractusx/tractusx-edc/assets/58386912/5c15a4cb-e4ad-48ab-96af-63699407ec77">

**Comment by florianrusch-zf** [comment]
Solved within https://github.com/eclipse-tractusx/tractusx-edc/pull/1305
